core/vm/interpreter.go

```diff
@@ -43,7 +43,7 @@ type Config struct {
 	JumpTable [256]operation
 
 	// Type of the EWASM interpreter
-	EWASMInterpreter string
+	EWASMInterpreter map[string]string
```

Comment:
The interpreter config should remain a string. There's no point in expanding it in `eth/backend`. Just split it where you actually need it (in the EWASM's constructor inside `core/vm`).